### PR TITLE
[#IOPAE-51] List migration status by Delegate

### PR DESCRIPTION
**Pull request by balanza**
Given a Delegate, retrieve a summary of claimed subscription ownership, grouped by Organization

### CODE REVIEW

File: GetMigrationsStatusByDelegate/__tests__/handler.test.ts [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+    );
+
+    expect(response.kind).toBe("IResponseSuccessJson");
+    const mockExpressResponse = createMockExpressResponse();
+    response.apply(mockExpressResponse);
+    expect(mockExpressResponse.status).toHaveBeenCalledWith(200);
+    expect(mockExpressResponse.json).toHaveBeenCalledWith({ items: [] });
+  });
+
+  it("should return success on retireved data", async () => {
```
Comment (LoFrance) [inline GetMigrationsStatusByDelegate/__tests__/handler.test.ts:118]: Typo: retrieved

File: utils/query.ts [hunk 2]
```diff
@@ -29,3 +29,26 @@ export const MigrationsByOrganization = (
     .and.where({ hasBeenVisibleOnce: true })
     // some subs have "deleted" in their name, we can skip them
     .and.not.whereILike("serviceName", "%deleted%");
+
+/**
+ * Query chunk filering migration table by a single organization
+ * Common filters are apllied, too
```
Comment (LoFrance) [inline utils/query.ts:35]: Typo: applied

File: GetMigrationsStatusByDelegate/handler.ts [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+export const createHandler = (config: IConfig, pool: Pool): Handler => async (
+  _context,
+  delegateId
+): ReturnType<Handler> =>
+  pipe(
+    getMigrationByDelegate(config, pool)(delegateId),
+    TE.mapLeft(e =>
+      ResponseErrorInternal(`Failed to execute query on database: ${e.message}`)
+    ),
+    TE.chainW(
```
Comment (LoFrance) [inline GetMigrationsStatusByDelegate/handler.ts:122]: Maybe we can use `chain` instead of `chainW`

**Review by LoFrance — CHANGES_REQUESTED** [review]

File: GetMigrationsStatusByDelegate/handler.ts (hunk 3 above)
Comment (balanza) [reply GetMigrationsStatusByDelegate/handler.ts:122]: good catch! thanks

**Pull request merged**